tests/test_cli_compile.py

```diff
@@ -312,7 +312,7 @@ def test_url_package(runner, line, dependency, generate_hashes):
             os.path.join(
                 MINIMAL_WHEELS_PATH, "small_fake_with_deps-0.1-py2.py3-none-any.whl"
             ),
-            "\nsmall-fake-a==",
+            "\nsmall-fake-a==0.1",
```

Comment:
Could we assert more strictly below in the test?
```python 
    assert rewritten_line in out.stderr.splitlines()
    assert dependency in out.stderr.splitlines()
```

Reply:
Done.